src/Util/EnvLoader.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * BigBlueButton open source conferencing system - https://www.bigbluebutton.org/.
+ *
+ * Copyright (c) 2016-2024 BigBlueButton Inc. and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with BigBlueButton; if not, see <https://www.gnu.org/licenses/>.
+ */
+
+namespace BigBlueButton\Util;
+
+use Dotenv\Dotenv;
+
+/**
+ * @see https://github.com/vlucas/phpdotenv
+ */
+class EnvLoader
```

Comment:
Loading environment file is the responsibility of the integrator, not the library. What can you suggest better? Can we have it in the tests dir? Is there any lightweght env lib that we can use instead?

Reply:
I am not sure what you mean? You don't like the place of the class in `src/Util` as it is just just in the tests? I can move it into the test-directory if you like to have it there. Or do you mean something else?